exercises/src/components/tags/assignment-type.jsx

```diff
@@ -3,22 +3,25 @@ import React from 'react';
 import { Icon } from 'shared';
 import { observer } from 'mobx-react';
 import Exercise from '../../models/exercises/exercise';
-import SingleDropdown from './single-dropdown';
+import MultiSelect from './multi-select';
 
 const TYPE = 'type';
-const TYPES = {
-  'conceptual-or-recall': 'Conceptual or Recall',
-  'conceptual': 'Conceptual',
-  'recall': 'Recall',
-  'practice': 'Practice',
-};
+const TYPES = [
+  { value: 'reading', label: 'Reading' },
+  { value: 'homework', label: 'Homework' },
+  { value: 'practice', label: 'Practice' },
+];
 
 function QuestionTypeTag(props) {
   const { isOpenEnded } = props.exercise;
   return (
-    <SingleDropdown
+    <MultiSelect
+      {...props}
       readonly={isOpenEnded}
-      {...props} label="Question Type" type={TYPE} choices={TYPES}
+      tagType="assignment-type"
+      label="Assignment Type"
+      type={TYPE}
```

Comment:
@nathanstitt I don't think `type` is used in the `MultiSelect` component.

Reply:
yep, its tagType for MultiSelect.  thanks!